lib/generators/rolemodel/css/base/templates/app/javascript/stylesheets/components/card.scss

```diff
@@ -5,3 +5,10 @@
   width: 100%;
   background: var(--color-white);
 }
+
+.card--table {
```

Comment:
I thought we were going to rename this to `card--no-padding` per our discussion in the frontend team

Reply:
You're right, my apologies. That comment must have gotten lost in my notes.